lte/gateway/c/oai/common/common_types.h

```diff
@@ -167,6 +167,13 @@ typedef enum {
   NAM_MAX,
 } network_access_mode_t;
 
+typedef struct supported_features_s {
+#define FEATURE_LIST_ID_2_EXTERNAL_IDENTIFIER
```

Comment:
how is this used?

Reply:
not yet used, I added for future usage if we require more features.